en/CONTRIBUTING.md

```diff
@@ -26,34 +26,33 @@
 
 - code blocks
 
-## Tables
+## Notes blocks
+- A single note:
+> **Note**:
+- Multiple notes:
+> **Notes**:
+> 1. some text
```

Comment:
1. Is it needed to capitalize the first letter of a sentence? some  -> Some?
2. Periods are required at the end of sentences.

Reply:
Thanks for the reminder.